core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractLibrary.java

```diff
@@ -131,7 +131,7 @@
   @JoinColumn(name = "lastModifier", nullable = false)
   private User lastModifier;
 
-  @Formula(value = "(SELECT MAX(lcl.changeTime) FROM LibraryChangeLog lcl WHERE lcl.libraryId = libraryId)")
+  @Formula(value = "(SELECT MAX(lmod.changeTime) FROM LibraryChangeLog lmod WHERE lmod.libraryId = libraryId)")
```

Comment:
I believe this currently has no effect since Libraries aren't loaded via Hibernate currently. When they are, the same pattern as is used for Samples will perform better. Is there any reason to keep the annotation here?